experimental/framework/fluid-static/src/types.ts

```diff
@@ -88,20 +91,14 @@ export interface IServiceAudienceEvents<M extends IMember> extends IEvent {
 export interface IServiceAudience<M extends IMember> extends IEventProvider<IServiceAudienceEvents<M>> {
     /**
      * Returns an map of all users currently in the Fluid session where key is the userId and the value is the
-     * member object
+     * member object.  The implementation may choose to exclude certain connections from the returned map.
```

Comment:
Perhaps we can add in a little details here of what connections are excluded
```suggestion
     * member object.  The implementation may choose to exclude certain, non-interactive connections from the returned map to represent the roster of live users.
```

Reply:
Will add a note on the default behavior.  I'm not sure we want to be as explicit about what is excluded beyond that connections could be excluded though